### DISCUSSION TITLE
关于自定义基建排班beta的问题

### DISCUSSION THREAD

**Discussion by Char-123** [MaaAssistantArknights/MaaAssistantArknights#2345] [Q&A] [answered]
试了试发现基建排班json文件里不能出现注释语句“//”，不然运行基建换班时会提示“出现未知错误”，这是正常特性嘛，要修bug或者在帮助文档里提示嘛？

**Comment by zzyyyl** [comment, marked as answer]
json 不支持注释语句：）

**Reply by Char-123** [reply]
这样嘛( ' ρ ')没写过json，示例文档里写了一大堆解释让我以为可以注释= =

**Comment by horror-proton** [comment]
经过扩展的 jsonc 支持注释, 但 MAA 用的 json 好像不能解析注释
